cmds/modules/provisiond/events.go

```diff
@@ -7,21 +7,20 @@ import (
 	"github.com/pkg/errors"
 	"github.com/rs/zerolog/log"
 	"github.com/threefoldtech/substrate-client"
-	"github.com/threefoldtech/zbus"
+	"github.com/threefoldtech/zos/pkg/events"
 	"github.com/threefoldtech/zos/pkg/gridtypes"
 	"github.com/threefoldtech/zos/pkg/provision"
-	"github.com/threefoldtech/zos/pkg/stubs"
 )
 
 type ContractEventHandler struct {
 	node   uint32
 	pool   substrate.Manager
 	engine provision.Engine
-	cl     zbus.Client
+	events *events.RedisConsumer
```

Comment:
shouldn't that be `rc` instead of events for readability?

Reply:
sure